src/diffusers/pipelines/flux/pipeline_flux_fill.py

```diff
@@ -627,6 +659,8 @@ def disable_vae_tiling(self):
     # Copied from diffusers.pipelines.flux.pipeline_flux.FluxPipeline.prepare_latents
```

Comment:
Can we copy from `FluxImg2ImgPipeline.prepare_latents` or `FluxInpaintPipeline.prepare_latents`?

Reply:
Sure , That would be more clean. Thanks for the review :)